jscomp/main/rescript_main.ml

```diff
@@ -142,6 +153,8 @@ let build_subcommand ~start  argv argv_len =
           ~package_kind:Toplevel 
           ~per_proj_dir:Bsb_global_paths.cwd
           ~forced:!force_regenerate in 
+      (** check if every dependency is there *)    
```

Comment:
`(** full build unless every dependency is already installed *)`

Reply:
In the check_deps_installed 
I added a more comprehensive doc to explain the heuristics employed